include/master_element/Hex8CVFEM.h

```diff
@@ -51,6 +51,7 @@ class HexSCV : public MasterElement
   using AlgTraits = AlgTraitsHex8;
 
   HexSCV();
+  KOKKOS_FUNCTION
   virtual ~HexSCV();
```

Comment:
If this is an empty body implementation in the C file we should probably move this to the header with 

```
virtual ~HexSCV() = default;
```

Reply:
Sure, I'll do that along with marking the constructors.